src/graph/entities/graph_entity.h

```diff
@@ -27,27 +27,38 @@ typedef GrB_Index EntityID;
 typedef GrB_Index NodeID;
 typedef GrB_Index EdgeID;
 
-typedef enum GraphEntityType {
-	GETYPE_NODE,
-	GETYPE_EDGE
+typedef enum
```

Comment:
Add comments explaining the purpose of this enum

Reply:
done